src/data/nav/pubsub.ts

```diff
@@ -22,36 +22,9 @@ export default {
       name: 'Getting started',
       pages: [
         {
-          name: 'JavaScript',
-          link: '/docs/getting-started/javascript',
-        },
-        {
-          name: 'React',
-          link: '/docs/getting-started/react',
-        },
-        {
-          name: 'React Native',
-          link: '/docs/getting-started/react-native',
-        },
-        {
-          name: 'Kotlin',
-          link: '/docs/getting-started/kotlin',
-        },
-        {
-          name: 'Swift',
-          link: '/docs/getting-started/swift',
-        },
-        {
-          name: 'Ruby',
-          link: '/docs/getting-started/ruby',
-        },
-        {
-          name: 'Python',
-          link: '/docs/getting-started/python',
-        },
-        {
-          name: 'Quickstart guide',
-          link: '/docs/getting-started/quickstart',
+          name: 'Overview',
```

Comment:
I think we need to call this something other than 'overview' so it's obvious this is where they are.

Reply:
Replaced with "Guides" for now.